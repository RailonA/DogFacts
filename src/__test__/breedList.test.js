import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import BreedList from '../containers/breedList';
import rootReducer from '../reducers/index';

describe('BreedList DOM', () => {
  it('renders correctly when store is provided', () => {
    const store = createStore(rootReducer);
    const breedList = renderer.create(
      <Provider store={store}>
        <BreedList />
      </Provider>,
    ).toJSON();
    expect(breedList).toMatchSnapshot();
  });
});
