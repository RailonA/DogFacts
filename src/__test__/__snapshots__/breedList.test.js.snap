// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreedList DOM renders correctly when store is provided 1`] = `
<div>
  <div>
    <div>
      <div
        className="searchWrapper d-flex  justify-content-center"
      >
        <select
          className="searchBar text-center "
          onChange={[Function]}
        />
      </div>
    </div>
    <div
      className=" d-flex container flex-wrap"
    >
      <div
        className="row "
      >
        <div
          className="d-flex flex-wrap  justify-content-center"
        />
      </div>
    </div>
  </div>
</div>
`;
